From tony Mon Sep 17 00:00:00 2001
From: Tony Lindgren <tony@atomide.com>
Date: Wed, 2 Jun 2021 20:39:07 +0300
Subject: [PATCH] Fix motorola power key that sends KEY_END instead of
 KEY_POWER

Somehow motorola stock android kernel v3.0.8 sends KEY_END instead
of KEY_POWER when the power key is pressed. Let's add that as a
select key. Otherwise kexecboot cannot be used on devices like
mz617 that have no physical keyboard.

Cc: Carl Philipp Klemm <philipp@uvos.xyz>
Signed-off-by: Tony Lindgren <tony@atomide.com>
---
 src/evdevs.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/evdevs.c b/src/evdevs.c
--- a/src/evdevs.c
+++ b/src/evdevs.c
@@ -371,6 +371,7 @@ int inputs_process_evdev(int fd)
 		case KEY_HENKAN:	/* Zaurus SL-6000 */
 		case 87:			/* Zaurus: OK (remove?) */
 		case 63:			/* Zaurus: Enter (remove?) */
+		case KEY_END:		/* Motorola Mapphone devices (107) */
 		case KEY_POWER:		/* GTA02: Power (116) */
 		case KEY_PHONE:		/* GTA02: AUX (169) */
 			action = A_SELECT;
-- 
2.31.1
